### PR TITLE
own: flesh out main documentation skeleton

### PR DESCRIPTION
**Pull request by leonore**
Putting up a PR (mostly @eseliger's work) to get us going on the Own main docs page.

## Test plan

`sg run docsite`

### CODE REVIEW

File: doc/own/index.md [hunk 1]
```diff
@@ -8,11 +8,72 @@
...
+- And set **Value** to **True**
+- Click **Create flag**
+
+## Concepts
+
+**Owner**: An owner is defined as a person or a team in Sourcegraph.
+
+A person can be ...
+
+A team is ...
```
Comment (eseliger) [inline doc/own/index.md:30]: we need to work on this more, but doesn't have to be here :D

Comment (leonore) [reply doc/own/index.md:30]: yep, that was my thinking- docs conflicts are just the worst so wanted to get it going

**Comment by eseliger** [comment]
This PR was not backported from what I can tell, is that right?

**Comment by leonore** [comment]
@eseliger no, I had a read through the [docs first](https://handbook.sourcegraph.com/departments/engineering/dev/tools/backport/#how-should-i-use-the-backporting-tool) and my understanding is you cherry-pick commits into your worktree to backport the changes. I thought we could do that with all docs follow up commits?

**Comment by eseliger** [comment]
ah maybe 😬 I thought we just had to add a backport label to the pr 🤔

**Comment by leonore** [comment]
oh yeah you are right- but we can do that after merge too. I'll add the label

**Pull request merged**